.travis.yml

```diff
@@ -3,12 +3,13 @@ cache:
   directories:
     - WcaOnRails/vendor/bundle
 rvm:
-  - 2.3.1
+  - 2.4.0
```

Comment:
Ah, interesting! After a little research, it looks like `.ruby_version` is just a RVM thing, which would not help out people who do not use RVM. It looks like Gemfile allows you to specify a ruby version, however, and RVM supports that instead of a `.ruby_version` as a way to specify the version of ruby your project uses (see https://rvm.io/workflow/projects#project-file-gemfile).

I'll try that out and see if travis picks it up.